package.json

```diff
@@ -38,7 +38,7 @@
   "homepage": "https://github.com/opensearch-project/opensearch-dashboards-functional-test",
   "dependencies": {
     "@cypress/skip-test": "^2.6.1",
-    "@opensearch-dashboards-test/opensearch-dashboards-test-library": "https://github.com/opensearch-project/opensearch-dashboards-test-library/archive/refs/tags/1.0.4.tar.gz",
+    "@opensearch-dashboards-test/opensearch-dashboards-test-library": "git+https://github.com/opensearch-project/opensearch-dashboards-test-library.git#main",
```

Comment:
@AMoo-Miki It seems @kavilla changed this a couple of weeks ago in this PR: https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/706, are you aware of that?

Reply:
And also we need to update `package-lock.json` as well.

Reply:
Yeah; I am torn between the argument for cache problems vs having to make a release to be able to use the tests.

My thought is that since we don't use these during development but rather use them in the workflows, it would be better to leave it to the devs to make sure they have the latest deps. That said, I haven't used either of these repos outside the workflows and maybe others do.

Thinking more about it, maybe we should stick to status quo and just bump to 1.0.5 and employ that.

Reply:
Raised https://github.com/opensearch-project/opensearch-dashboards-test-library/pull/42 for a release there and will update this PR to use 1.0.5.